bin/rector.php

```diff
@@ -124,10 +124,6 @@ public function loadIfExistsAndNotLoadedYet(string $filePath): void
 // load local php-parser only in prefixed version or development repository
 function should_include_preload(): bool
 {
-    if (file_exists(__DIR__ . '/../vendor/scoper-autoload.php')) {
```

Comment:
the `scoper-autoload.php` is exists in `phpstan-extracted/vendor/scoper-autoload.php`, move the check in : 

```php
$autoloadIncluder->loadIfExistsAndNotLoadedYet(__DIR__ . '/../vendor/phpstan/phpstan-extracted/vendor/scoper-autoload.php');
```